app/forms/providers/bank_statement_form.rb

```diff
@@ -94,10 +97,10 @@ def draft?
     # can be shared with v1 bank statement controller
     def create_attachment(file)
       attachment = legal_aid_application
-                     .attachments.create!(document: file,
-                                          attachment_type: "bank_statement_evidence",
-                                          original_filename: file.original_filename,
-                                          attachment_name: sequenced_attachment_name)
+                      .attachments.create!(document: file,
+                                           attachment_type: "bank_statement_evidence",
```

Comment:
nit: can we pass `@attachment_type` instead of the string  `bank_statement_evidence`